src/test/java/by/onliner/tests/smokeTests/CatalogSmokeTests.java

```diff
@@ -0,0 +1,32 @@
+package by.onliner.tests.smokeTests;
+
+import by.onliner.page.pages.CatalogPageItem;
+import by.onliner.page.pages.ShoppingCartPage;
+import by.onliner.tests.AbstractTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import static by.onliner.util.EqualsUtil.equalContains;
+
+public class CatalogSmokeTests extends AbstractTest {
+
+    @Test(groups = "positiveTests", description = "[Test-Case ID:ONL_002] Test for removing item from the shopping cart " +
+            "and checking the cart for emptiness")
+    public void removeItemFromShoppingCartTest() {
+        CatalogPageItem catalogPageItem = new CatalogPageItem();
+        ShoppingCartPage shoppingCartPage = new ShoppingCartPage();
+        mainPage.openPage();
+        mainPage.getMenu().waitForComponentOpened();
+        mainPage.openCatalogFirstItem();
+        catalogPageItem.addToCart();
+        mainPage.getMenu().openShoppingCartPage();
+
+        shoppingCartPage.removeItemFromCart();
+        String removedItemMessage = shoppingCartPage.getRemovedItemInformation();
+        String emptyCartMassage = shoppingCartPage.getEmptyCartMassage();
+
+        Assert.assertTrue(equalContains(removedItemMessage, "Вы удалили"), "The message \""
```

Comment:
for formatting strings please use String.format()

Reply:
Used